aptos-move/move-examples/upgrade-tutorial/step_1/sources/message_store.move

```diff
@@ -0,0 +1,59 @@
+module upgrade_tutorial::message_store {
```

Comment:
This looks similar to the existing message example we already have. Can we combine somehow?